src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough9.java

```diff
@@ -0,0 +1,26 @@
+/*
+FallThrough
+checkLastCaseGroup = true
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThrough9 {
+
+    void method5(int i, int j, boolean cond) {
+        while (true) {
+            switch (i) {
+                case 5: {
+                    i++;
+                }
+                    /* block */ /* fallthru */ // comment
+                case 6:
+                    i++;
+                    /* block */ /* fallthru */ // comment
```

Comment:
@nrmancuso,
we do have coverage for non last.

Reply:
> I would like to understand what the goal is here.

goal is to cover last case 100%, primary goal.
and looks like we able to cover not last too, we did not make very comprehensive coverage, but it is ok. This PR is already good step forward and we can create one more issue if we see some cases that are problematic to cover easily now.